axelrod/moran.py

```diff
@@ -107,7 +108,7 @@ def set_players(self):
             self.players.append(player)
         self.populations = [self.population_distribution()]
 
-    def mutate(self, index):
+    def mutate(self, index: int):
         """Mutate the player at index."""
         # Choose another strategy at random from the initial population
         r = random.random()
```

Comment:
@drvinceknight A full review for the PR wasn't done. I had missed many other types in the PR, hence the Thumbs down.